CONTRIBUTING.md

```diff
@@ -22,3 +22,12 @@ If you added a contributor by mistake, you can remove them in a comment with:
 
 If you are making a pull request on behalf of someone else but you had no part in designing the 
 feature, you can remove yourself with the above syntax.
+
+# Tests
```

Comment:
would you mind if I coped this same section for [`w3c/webvr`](https://github.com/w3c/webvr#readme)?

Reply:
That'd be awesome. We're trying to get all existing specs to adopt a similar policy, so the more the merrier :)